src/lint/rules/tests/indentation/closing_brace.rs

```diff
@@ -173,5 +202,14 @@ bank pcie_config {
 #[test]
 fn composite_indent_incorrect() {
     let rules = set_up();
-    assert_snippet(COMPOSITE_INDENT_INCORRECT, 4, &rules);
+    let expected_errors = define_expected_errors!(
+        RuleType::IN4,
+        (12, 12, 4, 5),
+        (12, 12, 2, 3),
+        (12, 12, 0, 1),
+        (11, 11, 16, 17),
+    ); // Order of expected errors is not intuitive here
```

Comment:
With a slightly more complicated comparison function, you could make the definition order-agnostic

Reply:
Done. I've used HashSet to compare the collections so they are order-agnostic